operator/README.md

```diff
@@ -14,6 +14,7 @@ Ensure you have JDK 11 (or newer) installed.
 Build the Docker image with:
 
 ```bash
+mvn clean install -pl :keycloak-operator-maven-plugin -am
```

Comment:
Just out of curiosity. Can't we install the Maven plugin and operator in a single step? It'd provide a better UX, IMHO.

Reply:
I will check, in general, for local development, we need to install the `maven-plugin` once and iterate with the operator build.

Reply:
I did a quick test (`mvn clean install` in `operator` module with local mvn repo cleaned of any traces of the Operator) and seemed to work fine. Maven seems smart enough to build the plugin dependency first.